src/Common/tests/Common.Tests.csproj

```diff
@@ -14,6 +14,10 @@
     <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
   </ItemGroup>
   <ItemGroup>
+    <ProjectReference Include="$(CommonTestPath)\System\Diagnostics\RemoteExecutorConsoleApp\RemoteExecutorConsoleApp.csproj">
+      <Project>{69e46a6f-9966-45a5-8945-2559fe337827}</Project>
```

Comment:
@tarekgh is correct here? I don't understand if there is a correct item group, DictionaryExtension.cs is here for instance.

Reply:
my suggestion is to move it to its own item group

  <ItemGroup>
    <ProjectReference Include="$(CommonTestPath)\System\Diagnostics\RemoteExecutorConsoleApp\RemoteExecutorConsoleApp.csproj">
      <Project>{69e46a6f-9966-45a5-8945-2559fe337827}</Project>
      <Name>RemoteExecutorConsoleApp</Name>
    </ProjectReference>
 <ItemGroup>

it is just preferences to not mix Compile items with other items. but this doesn't affect the net result anyway :-) 

---
In reply to: [187122686](https://github.com/dotnet/corefx/pull/29601#discussion_r187122686) [](ancestors = 187122686)